config/services.php

```diff
@@ -35,4 +35,10 @@
         'secret' => env('STRIPE_SECRET'),
     ],
 
+    'google' => [
+        'maps' => [
+            'key' => env('GOOGLE_MAPS_API_KEY')
```

Comment:
I created a separate guide so that we didn't clutter the main README. There is a link to it under `Requirements`.